### PR TITLE
attribution_sources.slt: Bump sleep times

### PR DESCRIPTION
**Pull request by def-**
Fixes: https://github.com/MaterializeInc/database-issues/issues/8824

### Checklist

- [ ] This PR has adequate test coverage / QA involvement has been duly considered. ([trigger-ci for additional test/nightly runs](https://trigger-ci.dev.materialize.com/))
- [ ] This PR has an associated up-to-date [design doc](https://github.com/MaterializeInc/materialize/blob/main/doc/developer/design/README.md), is a design doc ([template](https://github.com/MaterializeInc/materialize/blob/main/doc/developer/design/00000000_template.md)), or is sufficiently small to not require a design.
  
- [ ] If this PR evolves [an existing `$T ⇔ Proto$T` mapping](https://github.com/MaterializeInc/materialize/blob/main/doc/developer/command-and-response-binary-encoding.md) (possibly in a backwards-incompatible way), then it is tagged with a `T-proto` label.
- [ ] If this PR will require changes to cloud orchestration or tests, there is a companion cloud PR to account for those changes that is tagged with the release-blocker label ([example](https://github.com/MaterializeInc/cloud/pull/5021)).
  
- [ ] If this PR includes major [user-facing behavior changes](https://github.com/MaterializeInc/materialize/blob/main/doc/developer/guide-changes.md#what-changes-require-a-release-note), I have pinged the relevant PM to schedule a changelog post.

### CODE REVIEW

File: test/testdrive/attribution-sources.td [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+> CREATE TABLE u(x INT NOT NULL, y INT, z TEXT);
+
+> CREATE MATERIALIZED VIEW w AS
+  SELECT t1.x AS x, t1.z AS z1, t2.z AS z2
+  FROM u AS t1, u AS t2
+  WHERE t1.x = t2.y;
+
+# Sometimes id 19, sometimes 21
+# > SELECT id, global_id FROM mz_internal.mz_dataflow_global_ids ORDER BY id, global_id;
+# 21  UID
```
Comment (teskje) [inline test/testdrive/attribution-sources.td:88]: Probably depends on how often we had to retry previous queries. I think we should either remove the test or have a more generic check like the `count(*)` check we do below. Btw, above is another test that compares the contents of `mz_dataflow_global_ids` with a specific ID value, which is probably flaky in the same way.

**Review by teskje — APPROVED** [review]
Thanks!

**Comment by mgree** [comment]
I'd gently prefer an SLT, since it's rewritable (and I anticipate having more tests like this in the future).

Adding sleeps should work, but perhaps we could also just select fewer columns if we're concerned about different sources not all being ready at the same time?

Alternatively: what about an SLT command that says, "retry this query once every second, up to N times, until you get the result below"?

**Comment by def-** [comment]
> Alternatively: what about an SLT command that says, "retry this query once every second, up to N times, until you get the result below"?

(Or fixing the rewrite capability in testdrive)

**Comment by def-** [comment]
I have just bumped the sleep times for now, less effort and easier to maintain, and we don't lose the ids.

**Pull request merged**